app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,10 @@
+module OrdersHelper
+	def order_amount(od)
+		sum=0
+		 @orderitems= od.order_items.all
+		 @orderitems.each do |oi|
+		 	sum+=oi.menu_item.price
+		 end
+		 sum
+	end
```

Comment:
huge code smell here, you can use something like MenuItem.in(order_item_id: od.order_item_ids).sum(:price)